### DISCUSSION TITLE
Deploy alternative

### DISCUSSION THREAD

**Discussion by ghost** [TeamUltroid/Ultroid#269] [Ideas]
hi developer 😉
i want to share alternative to heroku
since heroku always ban user so please consider this as alternative

1. back4app.com
2. deploybot.com

**Comment by xditya** [comment]
It doesn't work. Already talked with them, they do not support python apps.

**Reply by ghost** [reply]
bruh 🗿🗿🗿🗿🗿

okay list updated
